paper-input.js

```diff
@@ -158,6 +158,14 @@ Polymer({
       label {
         pointer-events: none;
       }
+
+      label:before {
```

Comment:
Can you please move this inside `paper-input-container`? perhaps `::slotted(*::before)` (i don't remember if `::before` goes in or outside the parens)

Reply:
Done